units/event_log_tests.cc

```diff
@@ -69,7 +69,7 @@ TEST(Event, AttributeCreation) {
 }
 
 TEST(Event, CreateShaderModuleEventCreation) {
-  const int64_t timestamp_val = 1601314732230797664;
+  TimestampClock::time_point timestamp_val;
```

Comment:
Why isn't this initialized? What's the default value?

Reply:
Also below

Reply:
The default time_point is  00:00:00 - 1 January 1970. I can initialize all with `GetTimestamp()` function to be more clear.

Reply:
How about something in the middle: initialize it with the default, whatever it is: `TimestampClock::time_point x = {};`?

Reply:
Ok